config/env_sample.txt

```diff
@@ -0,0 +1,8 @@
+IPE_SHEET_ID=1fvYhtxlu3pa8vNBjOVzp1hwNBsv823mTD3uP6l5_WuI
+IPE_SHEET_NAME=Offering
+
+GSERVICE_ACCOUNT_PATH=secrets/ipe_gserviceaccount.json
+
+LOG_LEVEL=DEBUG
+
+IS_DOCKER_RUN=TRUE
```

Comment:
Need to change "IS_DOCKER_RUN=TRUE" to "IS_DOCKER_RUN=True", otherwise the TRUE value will fail the eval() function in get_ipe_data_frome_gsheets.py line 56:

 service_account_path: str = docker_run_path if eval(
            self.is_docker_run) else local_run_path

Reply:
Yeah.. I gave the instructions wrong. Now it should be fixed